### PR TITLE
project proposal

### PR DESCRIPTION
**Pull request by scarfacedotcom**

### CODE REVIEW

File: development-updates.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ With the gained insight into the protocol, the following weeks serve as a deep d
...
-| [Scar Face](https://github.com/scarfacedotcom)                 | [Update 3](https://hackmd.io/@0xScarFace/epfweek3)                                                                                   | [Update 4](https://hackmd.io/@0xScarFace/epfweek4b)                                                           |                                                                                                                                                                           |
+| [Scar Face](https://github.com/scarfacedotcom)                 | [Update 3](https://hackmd.io/@0xScarFace/epfweek3)                                                                                   | [Update 4](https://hackmd.io/@0xScarFace/epfweek4b)                                                           |  [Fuzzing Ethereum’s devp2p Protocol](https://hackmd.io/@0xScarFace/projectproposal)                                                                                                                                                                         |
```
Comment (taxmeifyoucan) [inline development-updates.md:125]: The link to project proposal should point to a document within repository

**Review by taxmeifyoucan — CHANGES_REQUESTED** [review]
Please point the link in dev updates doc to your file in `/projects`. 
One suggestion about the project proposal would be to add some links, e.g. to geth's devp2p implementation since I suppose your fuzzer will rely on it https://github.com/ethereum/go-ethereum/tree/master/cmd/devp2p

File: development-updates.md (hunk 1 above)
Comment (scarfacedotcom) [reply development-updates.md:144]: Done

Comment (scarfacedotcom) [reply development-updates.md:144]: Done

Comment (scarfacedotcom) [reply development-updates.md:144]: Done

**Comment by scarfacedotcom** [comment]
> Please point the link in dev updates doc to your file in `/projects`. One suggestion about the project proposal would be to add some links, e.g. to geth's devp2p implementation since I suppose your fuzzer will rely on it https://github.com/ethereum/go-ethereum/tree/master/cmd/devp2p

Done

**Pull request merged**